torch/onnx/experimental/fonnx/v1/exporter.py

```diff
@@ -0,0 +1,46 @@
+import io
+from typing import Optional
+
+import torch
+
+from .export_options import ExportOptions
+
+
+def dynamo_export(
+    torch_model: torch.nn.Module,
+    /,
+    *args,
+    options: Optional[ExportOptions] = None,
+    **kwargs,
+) -> ExportOutput:
+    """Export a torch.nn.Module to a ONNX graph
+
+    Args:
+        torch_model: User model to be exported
+        args: Positional inputs to `torch_model`
+        options: Advanced export options
+        kwargs: Keyword inputs to `torch_model`
+
+    Returns:
+        An in-memory representation of the exported ONNX model.
+        See ``ExportOutput`` for reference.
+    """
+    ...
+
+
+class ExportOutput:
+    """In-memory representation of a torch.nn.Module exported to ONNX
+
+    Args:
+        format: Format to save ONNX model."""
+
+    def __init__(self, format: onnx.ModelProto):
+        ...
```

Comment:
`format` → `onnx_model`? This needs to be accessible on the instance as well such that I can do something with `dynamo_export(Module()).onnx_model`?

Reply:
lgtm